src/panels/lovelace/hui-root.ts

```diff
@@ -282,9 +282,7 @@ class HUIRoot extends LitElement {
       haStyle,
       css`
         :host {
-          -ms-user-select: none;
-          -webkit-user-select: none;
-          -moz-user-select: none;
+          tabindex: -1,
```

Comment:
This is not the right fix.

Tabindex is actually an attribute, and from https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

> The tabindex global attribute indicates if its element can be focused, and if/where it participates in sequential keyboard navigation (usually with the Tab key, hence the name).
>
> A negative value (usually tabindex="-1") means that the element should not be reachable via sequential keyboard navigation, but could be focused with Javascript or visually. It's mostly useful to create accessible widgets with JavaScript. You can observe the #Tabindex_Accessibility example below to clear confusion.

Reply:
🤦‍♀️